src/index.ts

```diff
@@ -309,11 +309,13 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
         'Harness-SDK-Info': SDK_INFO
       }
 
-      const targetHeader = btoa(JSON.stringify(target))
+      if (globalThis.btoa) {
```

Comment:
What do we do if btoa isn't available?

Reply:
This bit of code sends the Target as a header in the request. If the API doesn't receive that, it looks it up. So not passing it just means we lose a perf optimisation. It's very easy to add `btoa` in manually and it may be an option for a specialised build of the React SDK for React Native.

Reply:
I'd approve but you already have an approval.